torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -1121,6 +1171,180 @@ glow::NodeValue PyTorchModelLoader::rescaleIntToUint(glow::NodeValue input) {
   }
 }
 
+Expected<NodeValue> PyTorchModelLoader::getQuantizedConvTransposeNode(
+    const torch::jit::Node *ptNode, const bool isRelu) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  const glow::TransposeNode *output;
+
+  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 8, outputs, 1));
+
+  // input
+  glow::NodeValue input;
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      input, getGlowNodeValueForValue(inputs[QuantizedConv2dInputs::input]));
+  input = rescaleUIntToInt(input);
+
+  input = F_.createTranspose("qconv_input_transposed", input, NCHW2NHWC);
+  glow::ShapeNHWC inputShape(input.dims());
+
+  // groups
+  glow::unsigned_t groups;
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      groups,
+      static_cast_expected<glow::unsigned_t>(iValToInt(
+          getGlowIValueForValue(inputs[QuantizedConv2dInputs::group]))));
+
+  // weight and bias
+  at::Tensor *ptTensor;
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      ptTensor, iValToPTTensor(getGlowIValueForValue(
+                    inputs[QuantizedConv2dInputs::packed_weights])));
+
+  auto op =
+      c10::Dispatcher::singleton().findSchema({"quantized::conv2d_unpack", ""});
+  CHECK(op.has_value());
+  auto unpackedParams = callOp(*op, *ptTensor);
+  const at::Tensor ptWeightTensor = unpackedParams[0].toTensor();
+  const c10::optional<at::Tensor> ptBiasTensor =
+      unpackedParams[1].toOptional<at::Tensor>();
+
+  // if groups == 1 it is regular conv
+  bool isGroupwiseQuantized = ptWeightTensor.is_quantized() &&
+                              ptWeightTensor.qscheme() == at::kPerChannelAffine;
+  if (isGroupwiseQuantized) {
+    isGroupwiseQuantized &= (groups > 1);
```

Comment:
isn't this the same without the if?

Reply:
you are right, this is some history stuff.